### DISCUSSION TITLE
Importing existing dropdown options

### DISCUSSION THREAD

**Discussion by simon1tan** [directus/directus#4927] [Feature Requests]
For dropdown choices, can we get an import from text file or a text field? Slow to enter a bunch. Value and option can be same.

**Comment by rijkvanzanten** [comment]
<img width="384" alt="CleanShot 2021-04-09 at 11 26 04@2x" src="https://user-images.githubusercontent.com/9141017/114203683-60fe0800-9926-11eb-9780-697f161f5b2a.png">

You can use the Raw Value option to copy/paste the JSON structure it saves 👍🏻

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
